api/admin.ts

```diff
@@ -156,3 +156,22 @@ export const getAampReportsByFormId = (form: number) : Promise<PendingAampReport
         requestedForm: form
     }
 });
+export const getReportChatMessages = (
+    report_id: number,
+    before: number,
+    after: number
+  ): Promise<TelegramMessageDetails[]> =>
+    request<TelegramMessageDetails[]>(true, {
+      method: "POST",
+      url: "/aamp/getReportChatMessages",
+      data: { report_id, before, after },
+    });
+  
+  export const getFullTranscriptMessages = (
+    report_id: number
+  ): Promise<TelegramMessageDetails[]> =>
```

Comment:
why is the line length limit SO short here? unless yall have been secretly coding from your phones let's make the first line function definitions one liners pls it hurts my eyes